src/constants/keyword.ts

```diff
@@ -12,6 +12,7 @@ export enum Keyword {
   MODEL = "model",
   SCHEMA = "schema",
   CONDITION = "condition",
+  IN = "in"
```

Comment:
Why is in here? It's not a keyword.

Reply:
Highlighting the CEL expression is a lower priority than not erroring out which is what we are currently doing.

And highlighting keywords anywhere because they may match part of a CEL expression is not right unless we only highlight it when in the CEL.

IMO in should be just part of the CEL expression, the default highlighting

Reply:
Moved `in` to generic symbols for CEL.